### ISSUE TITLE
Error during compiling compile.bat on ubuntu 20.04 jdk 11

### ISSUE THREAD

**Issue by ghost** [akeranen/the-one#98]
Got this error when executed sh compile.bat ob ubuntu
error: option -extdirs not allowed with target 11
Then i saw some issues where it was recommended to change -extdirs to -cp
then i got errors compiling.

**Comment by julianofischer** [comment]
Please, refer to #74

**Comment by ghost** [comment]
> Please, refer to #74

Yes i followed it just like another person got errors even i got them, and moreover in version 1.6 which was in the http://akeranen.github.io/the-one website there was no compile.sh file
If u have other version which works please provide the link.
Thank you.
